QueryExamples.md

```diff
@@ -0,0 +1,115 @@
+#### Query examples
+
+These queries show examples of problems (or non-problems) that can be identified with the graphql-analyzer.
+**Note: id values will need to be changed**
+
+### N+1
+**Description**
+Select the first ten posts on a user's wall, and get:
+- post -> body
+- author -> first name
+- author -> last name
+
+```
+{
+  posts(first: 10, receiver_id: 1) {
+    edges {
+      node {
+        id
+        author {
+          id
+          first_name
+          last_name
+        }
+        body
+      }
+    }
+  }
+}
+```
+
+### Indexed
+**Description**
+Select the comments for a specific post
+
+```
+{
+  comments(content_id: 1, content_type: "Post") {
+    edges{
+      node {
+        id
+      }
+    }
+  }
+}
+```
+
+### Non-Indexed (Full Table Scan)
+**Description**
+Perhaps we add to the GraphQL schema the ability to query for all the users with a certain first name.
+If we forget to add an index on the `users` table for `first_name`, we'll get a full table scan.
+
+```
+{
+  users(first_name: "Darrel") {
```

Comment:
This query seems to be broken?
`Field 'users' doesn't accept argument 'first_name'`

cc/ @justinkrol @DerekStride

Reply:
nvm this looks like it's working in prod. 
don't know why it's failing in dev for me

Reply:
*face palm* 
nvm lol, had to pull the latest from master in webscale